source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/EmissionValueKey.java

```diff
@@ -185,8 +185,64 @@ public void setSubstance(final Substance substance) {
     this.substance = substance;
   }
 
+  /**
+   * Converts this EmissionValueKey to a parseable string representation in the
+   * format "year:substance".
+   * This format is compatible with {@link #fromStringValue(String)} for
+   * serialization/deserialization.
+   * 
+   * @return String representation in format "year:substance"
+   * @see #fromStringValue(String)
+   */
+  @JsonKey
+  public String toStringValue() {
+    return year + ":" + substance.name();
+  }
+
+  /**
+   * Creates an EmissionValueKey from its string representation.
+   * Expects input in the format "year:substance" as produced by
+   * {@link #toStringValue()}.
+   * 
+   * @param value The string representation in format "year:substance"
+   * @return A new EmissionValueKey instance
+   * @throws IllegalArgumentException if the input is null, empty, or has invalid
+   *                                  format
+   * @throws NumberFormatException    if the year part cannot be parsed as an
+   *                                  integer
+   * @see #toStringValue()
+   */
+  @JsonCreator
+  public static EmissionValueKey fromStringValue(final String value) {
+    if (value == null || value.trim().isEmpty()) {
+      throw new IllegalArgumentException("Input value cannot be null or empty");
+    }
+
+    final String[] parts = value.split(":");
+    final String yearStr = parts.length > 0 ? parts[0].trim() : "";
+    final String substanceStr = parts.length > 1 ? parts[1].trim() : "";
+
+    if (yearStr.isEmpty() || substanceStr.isEmpty()) {
+      throw new IllegalArgumentException("Year and substance cannot be empty, got: '" + value + "'");
+    }
+
+    if (parts.length != 2) {
+      throw new IllegalArgumentException("Input must be in format 'year:substance', got: " + value);
+    }
+
+    try {
+      final int year = Integer.parseInt(yearStr);
+      final Substance substance = Substance.valueOf(substanceStr);
+      return new EmissionValueKey(year, substance);
+    } catch (NumberFormatException e) {
+      throw new NumberFormatException("Invalid year format: " + yearStr);
+    } catch (IllegalArgumentException e) {
+      throw new IllegalArgumentException("Invalid substance: " + substanceStr);
+    }
+  }
+
   @Override
   public String toString() {
-    return "EmissionValueKey [year=" + year + ", substance=" + substance + "]";
+    return "EmissionValueKey [year=" + year + ", substance=" + substance.name() + "]";
```

Comment:
think we normally use just `substance` in this case, using the normal `toString` of the enum?